kerl

```diff
@@ -1548,28 +1645,35 @@ do_install() {
 
     [ -n "$KERL_RELEASE_TARGET" ] && [ -f bin/cerl ] && cp bin/cerl "$absdir/bin"
 
-    list_add installations "$build_name $absdir"
+    if ! list_add installations "$build_name $absdir"; then
+        exit_install
+    fi
 
     emit_activate "$rel" "$build_name" "$absdir" >"$absdir"/activate
     emit_activate_fish "$rel" "$build_name" "$absdir" >"$absdir"/activate.fish
     emit_activate_csh "$rel" "$build_name" "$absdir" >"$absdir"/activate.csh
 
     if [ -n "$KERL_BUILD_DOCS" ]; then
+        cd "$ERL_TOP" || return 1
```

Comment:
This seems to have been missing (because of a previous `cd $absdir`)...

Reply:
ah yeah probably so